docs/custom-flows/embedded-email-links.mdx

```diff
@@ -1,46 +1,105 @@
+description: Learn how to build custom embeddable email link sign-in flows to increase user engagement and reduce drop off in transactional emails, SMS's, and more.
 ---
 
-# Embeddable email links
+# Embeddable email links with sign-in tokens
 
-An "email link" is a link, that when pressed, will automatically authenticate your user, so that they can quickly perform some action on your site with less friction than if they had to sign in manually.
+An "email link" is a link that, when visited, will automatically authenticate your user so that they can perform some action on your site with less friction than if they had to sign in manually. You can create email links with Clerk by generating a sign-in token, which you can detect and take action on in your frontend.
 
 Common use cases include:
 
-- Welcome emails, when users are added off a waitlist
-- Promotional emails for users
-- Recovering abandoned carts
-- Surveys or questionnaires
+- Welcome emails when users are added off a waitlist.
+- Promotional emails for users.
+- Recovering abandoned carts.
+- Surveys or questionnaires.
 
-## Creating email links
+This guide will teach you how to set this flow up.
 
-Embeddable email links leverage [sign-in tokens](https://clerk.com/docs/reference/backend-api/tag/Sign-in-Tokens#operation/CreateSignInToken) which can be created from your backend.
+<Steps>
+
+### Generate a sign-in token
+
+Embeddable email links leverage [sign-in tokens](https://clerk.com/docs/reference/backend-api/tag/Sign-in-Tokens#operation/CreateSignInToken), which are JWTs that can be used to sign in to an application without specifying any credentials. A sign-in token can be used at most once, and can be consumed from the Frontend API using the [`ticket`](/docs/references/javascript/sign-in/sign-in#sign-in-create-params) strategy.
+
+> [!NOTE]
+> By default, sign-in tokens expire in 30 days. You can optionally supply a different duration in seconds using the `expires_in_seconds` property.
+
+The following example demonstrates a cURL request that fetches a valid sign-in token: 
 
 ```bash
-curl --request POST \
-  --url 'https://api.clerk.com/v1/sign_in_tokens' \
-  -H 'Authorization: Bearer {{bapi}}' \
+curl 'https://api.clerk.com/v1/sign_in_tokens' \
+  -X POST \
+  -H 'Authorization: Bearer {{secret}}' \
   -H 'Content-Type: application/json' \
-  -d '{
-    "user_id": "user_28dVmJleKwFWrefTJN7skmrbtJi",
-  }'
+  -d '{ "user_id": "user_123" }'
 ```
 
-This will return a token, which can then be embedded as a query param in any link, something like:
+This will return a token, which can then be embedded as a query param in any link, such as the following example:
 
 `https://your-site.com/welcome?token=THE_TOKEN`
 
-Once you have the link, this can be embedded anywhere, such as an email.
+You can embedded this link anywhere, such as an email.
+
+### Detect tokens in email links
```

Comment:
I still think the title here should be something that accurately represents what they're going to do - first, they generated the token. now they are going to build a UI to handle signing in with the token.

"Detect tokens" is pretty vague in relation to what's actually happening in the section. And I know users are going to be searching for "how to sign in with a sign in token" so this suggestion would make it very clear to them that are going to see the flow for how to do that.

```suggestion
### Build a custom flow for signing in with a sign-in token
```